MvvmCross-iOSSupport/MvvmCross.iOS.Support.XamarinSidebar/MvxSidebarPresenter.cs

```diff
@@ -90,14 +104,25 @@ protected virtual bool ShowPanel(MvxPanelEnum panel, UIViewController viewContro
         {
             var navigationController = (SideBarViewController as MvxSidebarViewController).NavigationController;
 
+            if (navigationController == null)
+                return false;
+
             switch (panel)
             {
                 case MvxPanelEnum.Left:
                 case MvxPanelEnum.Right:
                     break;
                 case MvxPanelEnum.Center:
                 default:
-                    navigationController?.PushViewController(viewController, true);
+                    navigationController.PushViewController(viewController, true);
+                    break;
+                case MvxPanelEnum.CenterWithLeft:
+                    navigationController.PushViewController(viewController, true);
+                    viewController.ShowMenuButton(SideBarViewController as MvxSidebarViewController, true, false);
```

Comment:
Could you use parameter names here for the optional ones? It makes the code more defensive and easier to read. I mean:
`viewController.ShowMenuButton(SideBarViewController as MvxSidebarViewController, showLeft: true, showRight: false);`